dist/MoreMenu.js

```diff
@@ -576,6 +576,35 @@ $(function () {
     });
     return html;
   }
+  /**
+   * The #p-cactions element is always in the DOM, so we can use it
+   * to create our own menu without worrying about HTML or class changes.
+   *
+   * @param {string} parentKey
+   * @param {string} html of list items
+   * @throws {Error} if the element has been unexpectedly removed from the DOM
+   *  so that appropriate action can be taken.
+   * @return {string}
+   */
+
+
+  function buildMenuVector(parentKey, html) {
+    var $menu = $('#p-cactions').clone(true);
+
+    if ($menu.length) {
+      $menu.attr('id', "p-".concat(parentKey));
+      $menu.addClass(["mw-portlet-".concat(parentKey), "mm-".concat(parentKey), 'mm-tab']).removeClass('mw-portlet-cactions').attr('aria-labelledby', "p-".concat(parentKey, "-label"));
+      $menu.find('input[type="checkbox"]').attr('id', "p-".concat(parentKey, "-checkbox"));
+      $menu.find('label').attr('id', "p-".concat(parentKey, "-label"));
+      $menu.find('label').text(msg(parentKey));
+      $menu.find('ul').addClass('mm-menu').html(html);
+      return $menu[0].outerHTML;
+    } // Throwing an error will notify Wikimedia developers that they
+    // broke this script by showing up in our error logs.
+
+
+    throw new Error('MoreMenu: p-cactions is not in DOM. Expection with Vector maintaining team not met.');
```

Comment:
There's a typo at "Expection". Also I can't figure out the meaning of the second sentence. Consider re-wording.